### DISCUSSION TITLE
Iconify vs. Minimize

### DISCUSSION THREAD

**Discussion by MichaelWDenney** [kirill-grouchnikov/radiance#498] [Q&A] [answered]
It was noticed that the Windows controls in the ribbon interface are Maximize/Iconify rather than Maximize/Minimize.  Was there a reason to use Iconify and is there any means of overriding it?  (Our users are just more familiar with Minimize.)

**Comment by kirill-grouchnikov** [comment]
What's the difference between iconify and minimize? They are used interchangeably in `RadianceTitlePane` but both refer to the same action. And `RadianceRibbonTitleFramePane` doesn't have any special overrides for those control buttons.

Is this about the internal naming? The icon visuals? The tooltip shown when the mouse is over that button?

**Reply by MichaelWDenney** [reply]
No, difference that I am aware of.  As I said, our users are just more familiar with Minimize.
It is in the Windows icon menu and the minimize button tooltip:
![image](https://github.com/user-attachments/assets/d4c13839-edfd-40d3-98fb-f94d1d0e22c0)
![image](https://github.com/user-attachments/assets/4f3e4262-ca89-47a8-8b56-2380ec587fa9)

**Reply by kirill-grouchnikov** [reply, marked as answer]
There's a bit of inconsistency in core Swing look-and-feels as well

* https://github.com/openjdk/jdk/blob/master/src/java.desktop/share/classes/com/sun/swing/internal/plaf/basic/resources/basic.properties#L166 uses minimize
* But https://github.com/openjdk/jdk/blob/master/src/java.desktop/share/classes/com/sun/swing/internal/plaf/basic/resources/basic.properties#L212 uses iconify (for accessibility)
* https://github.com/openjdk/jdk/blob/master/src/java.desktop/share/classes/com/sun/swing/internal/plaf/metal/resources/metal.properties#L74 (Metal) uses minimize

I'm not opposed to changing Radiance's `SystemMenu.iconify` to "Minimize", but it would need to come with all the translations in https://github.com/kirill-grouchnikov/radiance/tree/sunshine/theming/src/main/resources/org/pushingpixels/radiance/theming/internal/resources. From a casual look, quite a few already treat the original as "Minimize" for translations.

Please file a bug to track this. Note that the work on https://github.com/kirill-grouchnikov/radiance/issues/400 will take about 2-3 months, and until then no bug fixes will be happening in the development version 8.0.

**Reply by MichaelWDenney** [reply]
See bug #499
